File: test/fixtures/extra.c
#include "fixture.h"

int llparse__print_off(llparse_t* s, const char* p, const char* endp) {
  llparse__print(p, endp, "");
  return 0;
}


int llparse__on_span(llparse_t* s, const char* p, const char* endp) {
  return llparse__print_span("span", p, endp);
}

File: test/fixtures/extra.ts
import { Buffer } from 'buffer';

export default (binding: any, inBench: boolean) => {
  const nop = () => 0;

  binding.llparse__print_off = inBench ? nop :
    (_: any, buf: Buffer, off: number) => {
      binding.llparse__print(off, '');
      return 0;
    };

  binding.llparse__on_span = inBench ? nop :
    (_: any, buf: Buffer, off: number, offLen: number) => {
      return binding.llparse__print_span('span', buf, off, offLen);
    };
};
